monkey/infection_monkey/exploit/tools/helpers.py

```diff
@@ -20,6 +22,13 @@ def get_target_monkey_by_os(is_windows, is_32bit):
     )
 
 
+def get_random_monkey_temp_bin() -> str:
+    character_set = list(string.ascii_letters + string.digits + "_" + "-")
+    safe_random = SystemRandom()
+    random_string = "".join(safe_random.choices(character_set, k=8))
+    return f"monkey_temp_bin_{random_string}"
```

Comment:
@ilija-lazoroski Nothing uses this besides powershell. I'd recommend changing it to `get_random_file_suffix()` and performing the string concatenation in the powershell exploiter. Other code will need the random file suffix (issue #1782) without the "monkey_temp_bin" part.